int64[3] joint_angles
int64[3] desired_pos
int64 feet_flag
---
int64[3] desired_joint_angles
